docs/kintoapps.md

```diff
@@ -93,11 +93,42 @@ Be sure to include in the authorization header. Example:
 
 To make a call to gRPC, talk to grpc.prod.kintohub.com with your gRPC service proto file, but include in the grpc-meta-data the authorization header as well.
 
+## Viewing KintoApp Logs
+
+You can access the last 100 logs of an environment your KintoApp is deployed in, by going to *Environments* via the short cut menu on your KintoApp card, or from the KintoApp Manage page. Once there, open the edit page, and select *View Logs* on the top right hand side.
+
+The logs will give you information on the following things: 
+- Response code ie: 200, 500 etc.
+- Duration of the call
+- Which endpoint was called
+- Which KintoBlock was called
+- Which version of the KintoBlock was called
+- The time and date of the call
+
+If you wish to view more details of the call, these can be found by expanding that particular calls row in the table. Here you can view the JSON of any logs you have made. 
+
+### Required Format for Console Logs
+
+In order to view Console Logs in our system, specific syntax must be used. Every call to the block will contain a header called `kinto-request-id`. This header will contain a value that needs to be included in the **Console Log** in order to associate each **Console Log** with the correct call. It is important to send your information formatted in JSON, with the correct quotation marks like this : `"key": "value"`. An example for a **Node.js / Express** app would look like this:
```

Comment:
"In our system to view console logs in our system" => your system?  Its also not clear when u say every call to the blcok will contain a haeder.

--
Console logs can help you easily debug what has occurred during the lifetime of an API call.  We require a specific json format because console logs will be able to be used for alerts and analytics in the near future.

//TODO image

All console logs that do not follow our format will not be available for you to view. The format is as follows:

{ "kinto_request_id": "acd12ff21-f2f21f1wf1-2f21f1s", "customdata": "hello", "message": "world" }

Every incoming API call that comes to your service will have a unique kinto_request_id. You must log that Id into the console log as the value for "kinto_request_id".  This allows us to track your specific call and perform magic!

Here's and example of what it would look like in nodejs

//here


You may use middleware to automatically do this for each call as well. We'll provide examples of this in the near future.